services/integration/src/routes/constitution/document.js

```diff
@@ -0,0 +1,8 @@
+const Router = require('express').Router();
```

Comment:
This files is not needed! We are generating all routes automatically. Please remove this file and all the associated directories.